DevProxy.Abstractions/Plugins/BasePlugin.cs

```diff
@@ -99,6 +99,15 @@ public TConfiguration Configuration
     }
     public IConfigurationSection ConfigurationSection { get; } = pluginConfigurationSection;
 
+    protected virtual IEnumerable<string>? GetConfigurationValue(string key, IEnumerable<string>? list, IEnumerable<string>? defaultList = default)

```

Comment:
Let's add a comment why we need this. We can reuse the comment we had in the code previously, in the `GraphMinimalPermissionsGuidancePlugin` class, line 64

Reply:
fixed